blog/Konferencja Tłumaczy 2023 - relacja/index.md

```diff
@@ -0,0 +1,317 @@
+
+### Trados i wtyczka Custom.MT
+
+Ciekawostką jest wtyczka Custom.MT do Tradosa, która jest podłączona do wielu różnych silników. I jeśli jest to ChatGPT, który korzysta z translatora DeepL, to, jak twierdzi Agenor, możemy wrzucić do DeepL’a swój słownik ze swoją terminologią, a potem możemy jeszcze to, co wygeneruje DeepL za pomocą prompta dodatkowo udoskonalić.
+
+### ChatGPT
+
+Tematem, który przewijał się przez całą prezentację był ChatGPT – jak działa, jak nim sterować, jak go uczyć, jakie ma źródła informacji (nie tylko Wikipedia), jak znaleźć idealne zapytanie dla czata, jak zwiększyć skuteczność zapytań, czy warto uczyć się „prompt engineering”.
+
+Było o łamaniu modelu (poprzez zapytania typu jak zbudować bombę, czy jak porwać kogoś dla okupu czy polecenia mu napisania wirusa). Jest to coraz trudniejsze, bo wprowadza się coraz więcej zabezpieczeń przed tego typu zapytaniami.
+Prelegent wspominał o ChatGPT na komórkę (można się z nim porozumiewać głosowo, nie trzeba pisać promptów), poruszył temat poufności ChatGPT (przechowywanie danych z promptów do celów nauki czata jest domyślnie wyłączone), przeglądania Internetu w ChatGPT. Mówił, że istnieje ponad 1000 pluginów do ChatGPT. Wspominał też o tym, że OpenAI udostępnia detektor tekstu pisanego – można sprawdzić czy tekst (np. oddawany przez kogoś jako oryginalna praca autorska) został napisany przez model.
+
+### Wpływ AI na rynek pracy
+
+„AI będzie potrzebować 10-15 lat, aby zastąpić dzisiejsze zawody. W tym czasie powstanie podobna liczba nowych.” (Yann LeCun, Chief AI Scientist, Meta)
+
+Prelegent mówił o tym, że wpływ AI na rynek pracy jest widoczny już dziś. Istnieje wiele raportów, które podają, jaki wpływ na rynek pracy ma lub będzie mieć sztuczna inteligencja.
+
+I tak, Goldman Sachs przewiduje, że na skutek popularyzacji AI pracę straci 300 milionów osób. Ale też jest gros osób, które pracy nie stracą, a będą korzystać z AI, żeby ją usprawnić. Prelegent podkreślał, że na rynku widoczny jest trend, że firmy wstrzymują duże rekrutacje, redukują zespoły, szczególnie w świecie IT. Trudno jest w tej chwili powiedzieć, na kogo firmy postawią, kto będzie korzystać z copilotów, wspomagaczy. Ale firmy będą stawiać raczej na doświadczone osoby, które będą w stanie ocenić rzetelność informacji, które modele wytwarzają.
+
+Ponadto według przeanalizowanych raportów około 18% pracy na świecie da się skomputeryzować. A efekty są już widoczne na rynkach rozwiniętych. A ponad 66% zawodów w USA jest podatne na wprowadzenie AI: 7% zostanie całkiem zastąpionych, 63% będzie pracować wspólnie z AI, 30% pozostanie bez zmian.
+
+### Rozwiązania AI są wszędzie
+
+•	Microsoft Copilot w aplikacjach Office
```

Comment:
Formatowanie listy lepiej osiągnąć przez użycie `- ` (myślnik a potem spacja) - aplikuje się do wszystkich list w dokumencie.

Reply:
Ok. To już wiem, czemu wypowiedzi zaczynane od myślnika przeskakiwały w bullet pointy.